src/Raven.Studio/wwwroot/Content/css/styles-common.less

```diff
@@ -431,8 +431,7 @@ i {
 .btn {
     [class*=" icon-"],
     [class^="icon-"] {
-        top: .1em;
-        transform: scale(1.1);
+        top: .2em;
```

Comment:
this is big change. Did you check few other icons? Does it look good?

Reply:
Yes, I've gone through most of the pages and the buttons looked fine. In fact, the icons look more center-aligned now. For reference, I've uploaded the comparison (based on Notification Center) here: https://issues.hibernatingrhinos.com/issue/RavenDB-17391/Create-new-icon-for-close-preview#focus=Comments-67-385789.0-0

Reply:
great thanks